06-statistical-testing.Rmd

```diff
@@ -543,7 +545,7 @@ chi_ex2_obs %>%
   gt(rowname_col = "TrustGovernment") %>%
   tab_stubhead(label = "Trust in Government") %>%
   tab_spanner(label = "Trust in People",
-              columns = everything())
+              columns = everything()) 
```

Comment:
Output far off page.  May need to do landscape if we want to keep it in this format.  Also should adjust it so 3 decimals across all items.